app/routes/people/person/personal-information/edit.js

```diff
@@ -22,16 +25,31 @@ export default class PeoplePersonPersonalInformationEditRoute extends Route {
     );
     const contacts = [];
     for (let position of positions) {
-      let address = await position.primaryContact.contactAddress;
+      let { primaryContact, secondaryContact, _position } = position;
+      if (!primaryContact) {
+        primaryContact = createPrimaryContact(this.store);
+        primaryContact.contactAddress = this.store.createRecord('address');
```

Comment:
It's a detail, but this line is a little bit redundant as we already create an address line 40 if we don't have one already ?

Reply:
true 👍🏼